src/components/date-picker/date-picker.tsx

```diff
@@ -259,6 +259,7 @@ export class DatePicker {
         setTimeout(() => {
             this.showPortal = false;
         });
+        this.textField.shadowRoot.querySelector('input').focus();
```

Comment:
this fixes the bug Sofie mentioned in #764. It also makes our tabbing work better as we no longer have focus on the root dom element after the datepicker has closed

Reply:
Looks good. Since we do it in `hideCalendar`, I assume this doesn't mean we set focus immediately on creation, but please make sure to check that @BregenzerK.